src/app/space/create/deployments/apps/deployment-card.component.ts

```diff
@@ -13,6 +13,12 @@ import { CpuStat } from '../models/cpu-stat';
 import { Environment } from '../models/environment';
 import { DeploymentsService } from '../services/deployments.service';
 
+const ICON_OK = 'pficon-ok';
+const ICON_WARN = 'pficon-warning-triangle-o';
+const ICON_ERR = 'pficon-error-circle-o';
+const MSG_OK = 'Everything is ok.';
+const STAT_THRESHOLD = .6;
+
```

Comment:
Maybe these can be made into StatusIcon and StatusMessage enums or something similar, so that the things are grouped together?